python-lib/symspell_checker.py

```diff
@@ -162,19 +162,24 @@ def symspell_check_word(self, word: AnyStr, language: AnyStr) -> Tuple[bool, Any
                 2. Corrected word if the word is misspelled and a correction if found,
                     else keep the original word
                 3. Spellchecker diagnosis string explaining the spellchecker action
+
         """
         (is_misspelled, correction, diagnosis) = (False, word, "")
         try:
+            cleaned_text = word.strip()  # re.sub(r"\W+", " ", word).strip()  # remove invisible characters
+            # if not cleaned_text:
```

Comment:
Delete comments ?

Reply:
Oops, good catch

Reply:
Deleted in e9b7f0ca8914a7ad8a58f22f4322d325da25f492